mobileapp/src/components/screens/notification/NotificationDetailScreen/NotificationDetailScreen.tsx

```diff
@@ -92,13 +110,63 @@ export const NotificationDetailScreen: React.FC<
     setErrorDialog(false)
   }
 
+  /**
+   * Return View with data from NotificationDetails
+   */
   return (
-    <Center justifyContent="center" h="100%">
+    <Center justifyContent="flex-start" h="100%">
       <LoadingModal processing={processing} />
       <ErrorAlert isOpen={errorDialog} close={alertClose} />
-      <Text>{notificationId}</Text>
-      <Button onPress={confirm}>confirm</Button>
-      <Button onPress={deny}>deny</Button>
+      <Text fontSize={20} fontWeight="bold">
+        {R.strings.details.id}
+      </Text>
+      <Text italic fontSize={17}>
+        {details?.id}
+      </Text>
+      <Text fontSize={20} fontWeight="bold">
+        {R.strings.details.type}
+      </Text>
+      <Text italic fontSize={17}>
+        {details?.type}
+      </Text>
+      <Text fontSize={20} fontWeight="bold">
+        {R.strings.details.message}
+      </Text>
+      <Text italic fontSize={17}>
+        {details?.message}
+      </Text>
+      <Text fontSize={20} fontWeight="bold">
+        {R.strings.details.address}
+      </Text>
+      <Text italic fontSize={17}>
+        {details?.address}
+      </Text>
+      <Text fontSize={20} fontWeight="bold">
+        {R.strings.details.amount}
+      </Text>
+      <Text italic fontSize={17}>
+        {details?.amount}
+      </Text>
+      <Text fontSize={20} fontWeight="bold">
+        {R.strings.details.status}
+      </Text>
+      <Text italic fontSize={17}>
+        {details?.status}
+      </Text>
+      <Stack direction="row" padding="15%" space="20">
+        <Button
```

Comment:
to już trochę pierdoła, ale moim zdaniem te guziki powinny być odwrotnie.

Reply:
zamienione